storage/connectors/bigquery/src/main/java/feast/storage/connectors/bigquery/compression/FeatureRowsBatch.java

```diff
@@ -132,13 +139,24 @@ private void initValues() {
   }
 
   private void toColumnar(Iterable<FeatureRowProto.FeatureRow> featureRows) {
+    int timestampColumnIdx = schema.getFieldCount() - 2;
```

Comment:
If someone change the schema ordering / add new fields, this part of the code will need to be changed. Otherwise the application might fail silently. Should we change this part such that it doesn't depend on schema ordering?